build/run.js

```diff
@@ -136,7 +136,7 @@ commands.build.rust = async function(argv) {
 
         console.log('Checking the resulting WASM size.')
         let stats = fss.statSync(paths.dist.wasm.mainOptGz)
-        let limit = 3.8
+        let limit = 3.83
```

Comment:
why such a big jump?

Reply:
Actually it is smaller than it seems, currently `main` fails to build because the limit is too low.
As there are no new dependencies, I don't really see anything we can reasonably do now to avoid this change. It seems to come directly from the code introduced (and perhaps how it affects the compression or sth…?)

Reply:
Ok, super, just asking because of introduced size guidelines.